apps/arweave/src/ar_block_pre_validator.erl

```diff
@@ -709,24 +687,11 @@ pre_validate_poa(B, PrevB, PartitionUpperBound, H0, H1, Peer) ->
 	RecallByte1 = RecallRange1Start + B#block.nonce * ?DATA_CHUNK_SIZE,
 	{BlockStart1, BlockEnd1, TXRoot1} = ar_block_index:get_block_bounds(RecallByte1),
 	BlockSize1 = BlockEnd1 - BlockStart1,
-	RebaseThreshold =
-		case B#block.height > ar_fork:height_2_7() of
-			true ->
-				PrevB#block.merkle_rebase_support_threshold;
-			false ->
-				case B#block.height == ar_fork:height_2_7() of
-					true ->
-						PrevB#block.weave_size;
-					false ->
-						infinity
-				end
-		end,
-	ArgCache = {BlockStart1, RecallByte1, TXRoot1, BlockSize1, ?STRICT_DATA_SPLIT_THRESHOLD,
-			{spora_2_6, B#block.reward_addr}, RebaseThreshold},
+	ArgCache = {BlockStart1, RecallByte1, TXRoot1, BlockSize1,
```

Comment:
Do we also have to update these lines to account for the ArgCache structure?

https://github.com/ArweaveTeam/arweave/pull/515/files#diff-c71b6858e1c10e9f544c7eb46e9f333f4d95f2afb7f2ee2e7eff5dbe1ba23871R409

https://github.com/ArweaveTeam/arweave/pull/515/files#diff-c71b6858e1c10e9f544c7eb46e9f333f4d95f2afb7f2ee2e7eff5dbe1ba23871L441

Reply:
The first 4 elements did not change so `insert_element(5, ...` results in the same order of the elements.

Reply:
got it. I guess this is why I'm not a fan of erlang tuple manipulation it always feels incredibly brittle and obfuscated to me. But it is what it is. Comment resolved!